dxr/plugins/rust/__init__.py

```diff
@@ -436,8 +436,13 @@ def process_csv(self, file_name, header_only):
                         print " - 'process_" + line[0] + "' not implemented!"
                         continue
 
-                    if 'file_name' in args and args['file_name'].startswith(self.tree.source_folder):
-                        args['file_name'] = args['file_name'][len(self.tree.source_folder)+1:]
+                    if 'file_name' in args:
+                        file_name_value = os.path.normpath(args['file_name'])
```

Comment:
I'm not sure if it does - the docs don't say anything about normalisation, so I'm not sure if it does that and if it does, if it can be relied on. It also doesn't say what would happen if start is not part of path - will it error out or return the absolute path? I specifically want the latter behaviour.